use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -1,27 +1,28 @@
 ---
-title: About data tiering
-excerpt: Learn how data tiering helps you save on storage costs
+title: About tiered storage
```

Comment:
I think there is a problem calling this section as "Tiered storage"
This entire section is about Tiering to S3 which is **one of many layers** in a  tiered storage architecture. This chapter is about Tiering data to Object storage. I think we need to make that clear.

Need to edit the content to reflect the multiple layers in storage and perhaps add the image from the blog doc (with different storage tiers) in the introduction section.
And then specifically call out that we provide special APIs to move data to object storage.